packages/twenty-server/src/engine/workspace-manager/workspace-migration-v2/interceptors/workspace-migration-builder-exception-v2-handler.ts

```diff
@@ -0,0 +1,25 @@
+import { t } from '@lingui/core/macro';
+
+import {
+  BaseGraphQLError,
+  ErrorCode,
+} from 'src/engine/core-modules/graphql/utils/graphql-errors.util';
+import { WorkspaceMigrationBuilderExceptionV2 } from 'src/engine/workspace-manager/workspace-migration-v2/exceptions/workspace-migration-builder-exception-v2';
+import { fromWorkspaceMigrationBuilderExceptionToValidationResponseError } from 'src/engine/workspace-manager/workspace-migration-v2/interceptors/utils/from-workspace-migration-builder-exception-to-validation-response-error.util';
+
+export const WorkspaceMigrationBuilderExceptionV2Handler = (error: Error) => {
```

Comment:
I'm not sure this is useful: handlers are specific to their resolver context. Their behavior would be different depending on what the user is trying to do